### PR TITLE
Jomenxiao/add recommend hardware

### PR DESCRIPTION
**Pull request by jomenxiao**
add recommend hardware

### CODE REVIEW

File: kafka-ansible/README.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ Default Version
 |Zookeeper|3.4.11|
 |Kafka|2.12-1.0.0|
 
+Recommend Hardware
```
Comment (LinuxGit) [inline kafka-ansible/README.md]: Recommended

File: kafka-ansible/README.md [hunk 2]
```diff
@@ -8,6 +8,13 @@ Default Version
...
+
+|Name|Cluster Size|Memory|CPU|Disk|
+|:---:|:---:|:---:|:---:|:---:|
+|Zookeeper|3+|16G|8+|2+ 1TB|
```
Comment (LinuxGit) [inline kafka-ansible/README.md]: zookeeper 1T?

File: kafka-ansible/README.md [hunk 3]
```diff
@@ -43,7 +50,9 @@ Do it
 	- `kafka_port` for client connect 
 	- `id` is the kafka's broker id. It must be set to a unique integer for each broker.
 	- one line mean one process
-	- example: `kafka1_1 ansible_host=172.17.8.201 deploy_dir=/home/tidb/kafka_deploy1 kafka_port=9091  id=1`
+	- kakfa's default data log directory is `$deploy_dir/datalog`
+	- more than one directory to stroge kafka's data; please setting `data_dirs` or **let it's NULL**
+	- example: `kafka1 ansible_host=172.17.8.201 deploy_dir=/home/tidb/kafka_deploy1 data_dirs=/home/tidb/kafka_deploy1,/tmp/kafka1 kafka_port=9091  id=1`
```
Comment (LinuxGit) [inline kafka-ansible/README.md]: we recommend /tmp/kafka1 ?

File: kafka-ansible/README.md (hunk 2 above)
Comment (jomenxiao) [reply kafka-ansible/README.md]: fixed it

File: kafka-ansible/zh文档.md [hunk 4]
```diff
@@ -12,8 +12,8 @@
 
 |名字|数量|内存|CPU|硬盘|
 |:---:|:---:|:---:|:---:|:---:|
-|Zookeeper|3+|16G|8+|2+ 1TB|
-|Kafka|3+|8G|4+|2+ 300G|
+|Kafka|3+|16G|8+|2+ 1TB|
+|Zookeeper|3+|8G|4+|2+ 300G|
```
Comment (LinuxGit) [inline kafka-ansible/zh文档.md]: zh文档.md need to update `data_dirs`

File: kafka-ansible/inventory.ini [hunk 5]
```diff
@@ -7,10 +7,12 @@ zk_3 ansible_host=172.17.8.203  deploy_dir=/home/tidb/zk_deploy myid=3
 # myid is zookeeper myid file, it's uniq; range 1-255
 # http://zookeeper.apache.org/doc/current/zookeeperAdmin.html#sc_configuration
 
+
+#https://www.ibm.com/support/knowledgecenter/en/SSPFMY_1.3.5/com.ibm.scala.doc/config/iwa_cnf_scldc_cnf_kfk_t.html
 [kafka_servers]
-kafka1 ansible_host=172.17.8.201 deploy_dir=/home/tidb/kafka_deploy1 kafka_port=9091  id=1
-kafka2 ansible_host=172.17.8.202 deploy_dir=/home/tidb/kafka_deploy1 kafka_port=9091  id=4
-kafka3 ansible_host=172.17.8.203 deploy_dir=/home/tidb/kafka_deploy1 kafka_port=9091  id=7
+kafka1 ansible_host=172.17.8.201 deploy_dir=/home/tidb/kafka_deploy1 data_dirs=/home/tidb/kafka_deploy1,/tmp/kafka1 kafka_port=9091  id=1
```
Comment (LinuxGit) [inline kafka-ansible/inventory.ini]: need to update

**Pull request merged**